Testnet/Ticketbooth/Ticketbooth/Ticketbooth.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Interesting 3.0 compiles in the contract tool. I wonder if they're even checking that. @rowandh or @codingupastorm might know. Will `netcoreapp3.x` contracts run fine even thought he FN is 2.x?

Reply:
I believe that validation will fail if we try and use 3.0 features in contracts, since features are whitelisted. Still interested to hear @rowandh and @codingupastorm take on this. For now I've switched to v2.1 just to be sure no issues can occur.